x-pack/plugins/gis/public/angular/map_controller.js

```diff
@@ -197,6 +233,10 @@ app.controller('GisMapController', ($scope, $route, config, kbnUrl, localStorage
     return { id };
   }
 
+  // Hide angular timepicer/refresh UI from top nav
+  timefilter.disableTimeRangeSelector();
+  timefilter.disableAutoRefreshSelector();
+  $scope.showDatePicker = true; // used by query-bar directive to enable timepikcer in query bar
```

Comment:
Is this ever `false`? Would it make sense to just hardcode `true` in the html file?